src/app/page.tsx

```
@@ -179,37 +194,145 @@ const MainPage = () => {
-                }
-              >
-                <MarketsSection
-                  markets={sortedAllMarkets as Market[]}
-                  handleSelectSort={handleSelectSort}
-                  isLoading={isFetching && !isFetchingNextPage}
-                  sort={selectedSort.sort}
+      <VStack w='full' spacing={0}>
+        {headerContent}
+
+        {selectedCategory?.name === 'Crypto' && !isMobile ? (
+          false ? (
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)